src/events.json

```diff
@@ -74,23 +74,85 @@
 		"hasInnerFill": true,
 		"gamePlayMode": "Comics"
 	},
+	"MidtermEvent": {
+		"prompt": "This midterm has an essay component ;_;",
+		"imgPath": "",
+		"choices": [
+			"LoremIpsum"
+		],
+		"hasBottomBoxBorder": false,
+		"hasInnerFill": true,
+		"gamePlayMode": "Minigame"
+	},
+	"CryMidtermPaperEvent": {
+		"prompt": "You failed your latest midterm...",
+		"imgPath": "CryMidtermPaperEvent/midtermPaper.png",
+		"choices": [
+			"DarnIt"
+		],
+		"hasBottomBoxBorder": false,
+		"hasInnerFill": true,
+		"gamePlayMode": "Comics"
+	},
+	"BlockPartyEvent": {
+		"prompt": "Block party is coming up !!!",
+		"imgPath": "BlockPartyEvent/blockParty.png",
+		"choices": [
+			"BlockPartyCancelled"
+		],
+		"hasBottomBoxBorder": false,
+		"hasInnerFill": true,
+		"gamePlayMode": "Comics"
+	},
+	"CoronaEvent": {
+		"prompt": "CORONA VIRUS 😳",
+		"imgPath": "CoronavirusEvent/corona.png",
+		"choices": [
+			"TikTok",
+			"ToiletPaper"
+		],
+		"hasBottomBoxBorder": true,
+		"hasInnerFill": true,
+		"gamePlayMode": "Comics"
+	},
 	"SquirrelEvent": {
 		"prompt": "You have a lecture across campus but there are squirrels in the way!",
 		"imgPath": "",
 		"choices": [
-			"UseWheels",
-			"SquirrelPictures"
+			"UseWheels"
```

Comment:
There should be another choice here.

Reply:
Let's leave the choice out so that the player has to play the minigame?

Reply:
aka don't give them an option to skip?

Reply:
sure that would work.